scripts/h5peditor-library-selector.js

```diff
@@ -247,7 +235,9 @@ ns.LibrarySelector.prototype.loadSemantics = function (library, params, metadata
   this.$selector.attr('disabled', true);
 
   ns.resetLoadedLibraries();
-  ns.loadLibrary(library, function (semantics) {
+  // ns.loadLibrary(library, function (semantics) {
+  ns.loadLibrary("H5P.InteractiveVideo 1.24", function (semantics) {
```

Comment:
IMHO not great to repeat the same string - including a chaning version number - so many times.

Reply:
Maybe i can create a variable that replaces the library!